incubator/prometheus-adapter/templates/custom-metrics-resource-reader-cluster-role.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "k8s-prometheus-adapter.name" . }}-resource-reader
+  labels:
+    app: {{ template "k8s-prometheus-adapter.name" . }}
+    chart: {{ template "k8s-prometheus-adapter.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  - pods
+  - services
+  - configmaps
```

Comment:
Is there a reason why you add configmaps? In the [original deployment files](https://github.com/DirectXMan12/k8s-prometheus-adapter/blob/master/deploy/manifests/custom-metrics-resource-reader-cluster-role.yaml) are only `namespaces`, `pods` and `services` listed.

Reply:
@florianrusch in my testing I ran into the following problem when trying to start it without configmaps.
```
panic: configmaps "extension-apiserver-authentication" is forbidden: User "system:serviceaccount:application-monitoring:prometheus-adapter-prometheus-adapter" cannot get configmaps in the namespace "kube-system": Unknown user "system:serviceaccount:application-monitoring:prometheus-adapter-prometheus-adapter"
```